enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,983 @@
+     loadBalancer:
+       dnsManagementPolicy: Managed
+       providerParameters:
+         aws:
+           networkLoadBalancer:
+             eipAllocations:
+               - eipalloc-0956fea34de4cb7ab
+               - eipalloc-0e9a3077a70de050a
+               - eipalloc-0b69fc4691f54cdd0
+               - eipalloc-01e6ba6cbba1a391b
+               - eipalloc-0e242df173f906112
+           type: NLB
+         type: AWS
+       scope: External
+     type: LoadBalancerService
+...
+
+```
+
+#### Cluster admin wants operator to automatically effectuate the Load Balancer service for updated Ingress Controller eipAllocations 
+
+The administrator can tell the Ingress Operator to complete the disruptive update of the service automatically
+by annotating the IngressController with the annotation called `ingress.operator.openshift.io/auto-delete-load-balancer` 
+and then changing the eipAllocations by using the following commands:
+
+Annotate ingress controller with `ingress.operator.openshift.io/auto-delete-load-balancer` with a blank value.
+```sh
+oc -n openshift-ingress-operator annotate ingresscontrollers/test ingress.operator.openshift.io/auto-delete-load-balancer=
+```
+
+Update the Ingress Controller with the following command:
+
+```sh
+% oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"type":"LoadBalancerService","loadBalancer":{"providerParameters":{"type":"AWS","aws":{"type":"NLB","nlbParameters":{"eipAllocations":[]}}}}}}}'
+```
+
+After the changes are saved, the operator will delete and create the load balancer service automatically.
+```sh
+% oc get svc router-test -n openshift-ingress 
+NAME          TYPE           CLUSTER-IP       EXTERNAL-IP                                                                     PORT(S)                      AGE
+router-test   LoadBalancer   172.30.230.109   a95d070adbbf045c9b356ed9db8d3cda-89fbf4dfc638408e.elb.us-east-1.amazonaws.com   80:30453/TCP,443:32704/TCP   77s
+```
+
+If you remove all EIPs, the annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations` is removed from the service as well.
+
+
+#### Unmanaged Subnet Annotation Migration Workflow
+
+Migrating an unmanaged `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`
+service annotation to `spec.endpointPublishingStrategy.loadBalancer.providerParameters.aws.networkLoadBalancerParameters.eipAllocations`
+after upgrading to 4.y doesn't require a cluster admin to delete the LoadBalancer-type
+service:
+
+1. Cluster admin confirms that initially `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`
+   is set on the LoadBalancer-type service managed by the IngressController.
+2. The cluster admin upgrades OpenShift to v4.y and the service annotation is not
+   changed or removed. For more details see [section](#Any change to the service annotation does not trigger the operator to manage the load balancer service).  
```

Comment:
`[section]` This doesn't render as a link. It may be because you have the wrong link (I think you are missing a `4. `

Reply:
yes updated.